generator/cmd/postprocessor.ts

```diff
@@ -0,0 +1,66 @@
+import path from 'path';
+import chalk from 'chalk';
+import { executeSynchronous, lowerCaseEquals } from '../utils';
+import { findAutogenEntries } from '../autogenlist';
+import { saveAutogeneratedSchemaRefs, SchemaConfiguration, schemaPostProcess } from '../generate';
+import * as constants from '../constants';
+
+async function getChangedSchemas(repoPath: string) {
+    var git = require('nodegit-kit');
+    const changedSchemas = [];
+    const repo = await git.open(repoPath);
+    const status = await git.status(repo);
+    for (const stat of status) {
+        if (path.basename(path.dirname(path.dirname(stat.path))) === 'schemas'
+            && path.extname(stat.path) === '.json') {
+            if (stat.status !== 'modified' && stat.status !== 'new') {
+                throw new Error(`Undefined file status for '${stat.path}'.`);
+            } else {
+                if (stat.status === 'modified') {
+                    changedSchemas.push({
+                        'path': path.join(repoPath, stat.path),
+                        'isNew': false
```

Comment:
why do you use single quote here? can you define the array element type explicitly?